addons/web/static/tests/views/abstract_view_tests.js

```diff
@@ -71,21 +70,20 @@ QUnit.module('Views', {
                 assert.step(url + ' loaded');
             });
         };
+
         var View = AbstractView.extend({
-            config: _.extend({}, AbstractView.prototype.config, {
-                js_libs: [
-                    ['a', 'b'],
-                    ['c'],
-                ],
-            }),
+            jsLibs: [
+                ['a', 'b'],
+                'c',
+            ],
         });
         createAsyncView({
             View: View,
             arch: '<fake/>',
             data: this.data,
             model: 'fake_model',
         }).then(function (view) {
-            assert.verifySteps(['a', 'b', 'a loaded', 'b loaded', 'c', 'c loaded'],
+            assert.verifySteps(['a', 'b', 'b loaded', 'a loaded', 'c', 'c loaded'],
```

Comment:
I change the load order for the test (resolve b before a) to check that is parallel. Also not the test would pass if the list in the list is also sequential